EXILED/Exiled.API/Features/Map.cs

```diff
@@ -417,6 +424,19 @@ public static void PlayGunSound(Vector3 position, ItemType firearmType, byte max
             msg.SendToAuthenticated();
         }
 
+        /// <summary>
+        /// Spawns a Mice inside the <see cref="RoomType.EzShelter"/>.
+        /// </summary>
+        /// <param name="mice">The type of mice you want to spawn..</param>
+        public static void SpawnMice(byte mice = 1)
+        {
+            if (mice > SqueakSpawner.mice.Length)
+                new OverflowException($"The mice type are from 1 to {SqueakSpawner.mice.Length}");
```

Comment:
```suggestion
                new ArgumentOutOfRangeException($"The mice type are from 1 to {SqueakSpawner.mice.Length}");
```

Do you really want to use exception here? You can use just `Log.Error`. 
Also, just creating a new instance of exception won't stop execution of method